client/app/home/board/invite/invite.js

```diff
@@ -0,0 +1,32 @@
+angular.module('artemis.board.invite', [])
+.controller('InviteController', function($scope, $state, $stateParams, Boards, Users) {
+  $scope.boardId = $stateParams.id;
+  $scope.ACL;
+
+  $scope.submit = function() {
+    var email = $scope.invitee;
+    var userId;
+    Users.getUserIdFromEmail(email).then(function(email) {
+      userId = email;
+      $scope.getACL(userId);
+    });
+  };
+
+  $scope.getACL = function(userId) {
+    Boards.getBoard($scope.boardId).then(function(board) {
+      $scope.ACL = board.ACL;
+      $scope.addToACL(userId);
+    });
+  };
+
+  $scope.addToACL = function(userId) {
+    var boardId = $scope.boardId;
+    var ACL = $scope.ACL;
+    var token = Users.getSessionToken();
+    Boards.addToACL(userId, boardId, token, ACL).then(null, null);
```

Comment:
You can just take out the `.then(null, null)` if you don't need to do anything after the promise is resolved/rejected